src/Microsoft.Unity.Analyzers.Mpack/AddinInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Mono.Addins;
+using Mono.Addins.Description;
+
+[assembly: Addin(
+	"Microsoft.Unity.Analyzers",
+	Category = "Game Development"
+)]
+
+[assembly: AddinName("Microsoft.Unity.Analyzers")]
+[assembly: AddinCategory("IDE extensions")]
+[assembly: AddinDescription("Roslyn Analyzers for Unity developers")]
+[assembly: AddinAuthor("Microsoft Corporation")]
+
+[assembly: AddinDependency("MonoDevelop.Core", MonoDevelop.BuildInfo.Version)]
+[assembly: AddinDependency("MonoDevelop.Ide", MonoDevelop.BuildInfo.Version)]
+[assembly: AddinDependency("MonoDevelop.CSharpBinding", MonoDevelop.BuildInfo.Version)]
```

Comment:
Additional dependencies on MonoDevelop.Refactoring and MonoDevelop.TextEditor are required.